packages/i18n/src/locales/en.i18n.json

```diff
@@ -3407,6 +3407,8 @@
   "manage-livechat-units_description": "Permission to manage omnichannel units",
   "manage-oauth-apps": "Manage OAuth Apps",
   "manage-oauth-apps_description": "Permission to manage the server OAuth apps",
+  "view-oauth-apps_description": "Permission to view the server OAuth apps",
+  "view-oauth-apps": "View OAuth Apps",
```

Comment:
I'm wondering if we can have another name for this setting so as to make it clear that this is needed for users to login. I'll check this internally and bring some suggestion later

Reply:
Any suggestions @matheusbsilva137 ?